packages/sdk/src/types/relationshipTemplates/ConnectorRelationshipTemplate.ts

```diff
@@ -7,7 +7,7 @@ export interface ConnectorRelationshipTemplate {
     createdBy: string;
     createdByDevice: string;
     createdAt: string;
-    content: ConnectorRelationshipTemplateContentDerivation;
+    content: RelationshipTemplateContentJSON | ArbitraryRelationshipTemplateContentJSON;
```

Comment:
We could still define `RelationshipTemplateContentDerivation = RelationshipTemplateContentJSON | ArbitraryRelationshipTemplateContentJSON`, since it is used in multiple places.

Reply:
see above